### ISSUE TITLE
Progress Bar on UPLOAD step of importer modal is gone

### ISSUE THREAD

**Issue by jonpmarino** [MapStory/mapstory#948]
There used to be a progress bar during the Upload step of the Importer Modal. On BETA its no longer there. The user is still prevented from hitting the Next Step button until the Upload is complete however. Without a progress bar, it seems to a user like the Next Step button is just not working, and the user has no way of knowing that their Upload is actually in progress.

![screen shot 2017-08-14 at 12 01 17 pm](https://user-images.githubusercontent.com/2573761/29283367-0f3f5f42-80ec-11e7-9bdc-21176228c5c5.jpg)

**Comment by emilyashley** [comment]
Where were we getting the layer.state from anyhow?

The template doesn't appear to be modified, but the "UPLOADED" layer state doesn't seem to be in scope, and the uploadInProgress has a #todo next to it.

**Comment by davisc** [comment]
@jonpmarino  fixed in https://github.com/GeoNode/django-osgeo-importer/blob/master/osgeo_importer/static/osgeo_importer/importer.js

@cuttlefish may need to bump importer to this commit.

This snippet previously existed in the mapstory branch of importer as indicated by the version deployed to mapstory.org

<img width="913" alt="screen shot 2017-09-13 at 8 13 54 pm" src="https://user-images.githubusercontent.com/947403/30407801-24379856-98c0-11e7-96a6-7c8e6aa03faf.png">

**Comment by emilyashley** [comment]
confirmed this was bumped in https://github.com/MapStory/mapstory/commit/cce50b976eb42499408678a0584b122d6a4dedf0

**Comment by emilyashley** [comment]
confirmed locally: ![Screen Shot 2017-09-18 at 3.38.31 PM.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/570697024a4dff0e00dfbeed/125516c66e82c728ace21e0d46db978826878de9bce1b057e40fa0c57e0b772a6455fb6b7191a37bf111234f5c5e5ecc3c4a4356bcbbd671f3e57862804e0ee5d6695e73c03d40abbd804dbb02eb208b4d86beb4de1ad2f42f67b6d5db1063474f4dcbed5727a19141c95df64ee64b52a5103761e4712431.png)

**Comment by jonpmarino** [comment]
Reopening for the backlog. the "Great, We've got your data!" info is different from the progress bar that used to exist and was the purpose of this issue. The progress bar showed a a percentage advancing as your dataset uploaded to let you know how much time was left.  For example, in the screenshot I've uploaded a layer with 81,000 features. Right now there is no indicator to give me a sense of how long this upload will take. As a user, you're not sure if something is happening and, if so, how far along things are. So, as a user I click "next step" and nothing happens so I assume something is broken when actually the upload is still in process.
![screen shot 2018-06-11 at 4 42 14 pm](https://user-images.githubusercontent.com/2573761/41258710-cf55d6e8-6d96-11e8-9036-977793d10497.jpg)

If the percentage feature can't be re-implemented, at least just adding the spinning wheel that we already have for the "import" phase once the layer is "uploaded" would help. That at least tells the user a process is active. Here's a screenshot of the spinning wheel in action:
![screen shot 2018-06-11 at 4 47 35 pm](https://user-images.githubusercontent.com/2573761/41258895-6436cb6e-6d97-11e8-9814-d880befbd21e.jpg)
